bulk-import-tools/internal/orchcli/orchcli.go

```diff
@@ -410,6 +463,9 @@ func (oC *OrchCli) AllocateHostToSiteAndAddMetadata(ctx context.Context, hostID,
 	}
 	// Prepare the form data
 	payload := &api.Host{}
+	if host, ok := oC.HostCache[hostID]; ok {
+		payload.Name = host.Name
```

Comment:
@pierventre should we fix this once and for all in API?

Reply:
is there already a ticket for the issue?

Reply:
I have a vague memory of the issue - did we discuss a possible fix ?

Reply:
@rranjan3 please, confirm. But I believe this is done because we require the Name to be set in a PATCH, right?

Reply:
Yes. That was why I had to retain the host & ensure name is carried over. It's a required field in the API in the absence of which the name is overwritten with an empty string.

Reply:
@raphaelvrosa can you make sure this is fixed in apiv2?

Reply:
let's skip the fix for APIv1 @rranjan3

Reply:
Done in https://github.com/open-edge-platform/infra-core/pull/97/commits/78e68826a58efaea03f950a6d3389983d17013e9